test/test_utils.h

```diff
@@ -104,15 +104,16 @@ namespace test_utils {
 
 				const detail::command_id cid = pkg.cid;
 				commands[cid] = {nid, pkg, dependencies};
-				if(pkg.cmd == detail::command_type::TASK) { by_task[std::get<detail::task_data>(pkg.data).tid].insert(cid); }
+				if(pkg.cmd == detail::command_type::EXECUTION) { by_task[std::get<detail::execution_data>(pkg.data).tid].insert(cid); }
+				if(pkg.cmd == detail::command_type::HORIZON) { by_task[std::get<detail::horizon_data>(pkg.data).tid].insert(cid); }
```

Comment:
And this does not break any tests? Interesting.

Reply:
`by_task` is only used when passing a `task_id` to `cdag_inpector::get_commands()`, which was never done for horizon tasks as far as I'm aware.